src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,14 @@
 Release History
 ===============
 
+**IoT Central**
```

Comment:
It appears most of these fixes/changes are a function of updating the SDK. Is that right?

Reply:
Yes, not much changes we need to do on az cli side other than updating the SDK except the etag.